README.md

```diff
@@ -0,0 +1,27 @@
+# AI-IaC-Prompts
```

Comment:
I would spell out/explain more what "IaC" means - this acronym may not be familiar to everyone.